README.md

```diff
@@ -298,33 +298,33 @@ for(int i=0;i<rs.totalCount();)
 }
 ```
 
-### 3.1.4 Disconnect from the Maximo
+### 3.1.4 Disconnect from Maximo Asset Management
 
-* End the session with Maximo after you finish the work
+* End the session with Maximo Asset Management after you finish the work:
 
 ```java
 mc.disconnect();
 ``` 
 
-## 3.2 Create a new Work Order (mxwodetail)
+## 3.2 Create a new work order (mxwodetail)
 The following instruction shows how to create a new work order by Maximo Rest Client.
-### 3.2.1 Get the Work Order Set
+### 3.2.1 Get the work order set
 
-* Connect to the Maximo, 
+* Connect to Maximo Asset Management  
```

Comment:
line 313 
It occurs to me that Connect to Maximo is the first item on every thing and that it also has its own separate section. Recommend putting Connecting to Maximo as a prereq to eliminate the need to mention the connection every time. It's clear that you need to connect to Maximo first. Having the same info repeated in the document increases the chances for error if something ever changes and also makes the document way longer (and look more complicated) than it needs to be.